plotter/invertedaxis_example_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright ©2017 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package plotter_test
+
+import (
+	"image/color"
+	"log"
+	"math"
+	"testing"
+
+	"gonum.org/v1/plot"
+	"gonum.org/v1/plot/internal/cmpimg"
+	"gonum.org/v1/plot/plotter"
+	"gonum.org/v1/plot/vg"
+)
+
+// Example_invertedScale shows how to create a plot with an inverted Y-axis.
+// This is nearly identical to the Log example, except it inverts the Y axis.
+func Example_invertedScale() {
+	p, err := plot.New()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	p.Title.Text = "Exampled of an Inverted Axis"
```

Comment:
`s/Exampled/Example/`
`s/an Inverted Axis/inverted axes/` (as both x and y are inverted)